src/main/java/bio/terra/stairway/FlightMap.java

```diff
@@ -118,7 +118,7 @@ public String toString() {
         map.entrySet().stream()
             .map(
                 entry -> {
-                  String valString = entry.getValue().toString();
+                  String valString = String.valueOf(entry.getValue());
```

Comment:
will that return a null if entry.getValue() is null?

Reply:
It will return `"null"`

Reply:
Ah got it.  thanks

Reply:
I don't think we actually want to add this test, but this shows the behavior:
https://github.com/DataBiosphere/jade-data-repo/compare/sh-add-string-test?expand=1